evil-ex.el

```diff
@@ -242,7 +248,8 @@ interactive actions during ex state."
   (with-no-warnings
     (make-variable-buffer-local 'completion-at-point-functions))
```

Comment:
As a side note, why `make-variable-buffer-local` instead of `setq-local` or `(set (make-local-variable 'completion-at-point-functions))`?